draft-ietf-capport-architecture.xml

```diff
@@ -112,24 +112,40 @@
        principles:
       </t>
       <ul spacing="normal">
-        <li>Solutions SHOULD NOT require the forging of responses from DNS or
-            HTTP servers, or any other protocol.  In particular, solutions
-            SHOULD NOT require man-in-the-middle proxy of TLS traffic.</li>
-        <li>Solutions MUST operate at the layer of Internet Protocol (IP) or
-            above, not being specific to any particular access technology such
-            as Cable, WiFi or mobile telecom.</li>
-        <li>Solutions MAY allow a device to be alerted that it is in a
-            captive network when attempting to use any application on the
-            network.</li>
+        <li>Current captive portal solutions typically implement some variations
+            of forging DNS or HTTP responses.
+            Some reportedly perform man-in-the-middle (MITM) proxy of HTTPS in
+            order to forge reponses.
+            Captive Portal Solutions should not have to break any protocols or
+            otherwise act in the manner of an attacker.
+            Therefore, solutions MUST NOT require the forging of responses from
+            DNS or HTTP servers, or any other protocol; solutions MUST NOT
+            require man-in-the-middle proxy of TLS traffic.</li>
```

Comment:
```suggestion
            DNS or HTTP servers, or any other protocol.</li>
```
I think that use of "forging" is sufficient here.

Reply:
FWIW, as part of #103, we had some suggestions to rephrase this specific point to become a positive, rather than negative requirement. I think this section deals with the original objection well enough by laying out why it's a problem, but Michael's suggestion may still improve things.

Reply:
I'm going to break into two points: 1. don't forge responses; 2. the user should be able to use DNSSEC and TLS (Michael's wording.)